decidim-proposals/spec/services/decidim/proposals/proposal_serializer_spec.rb

```diff
@@ -148,6 +149,25 @@ module Proposals
             expect(serialized).to include(answer: expected_answer)
           end
         end
+
+        context "with rich text proposal body" do
+          let(:image) { "<img src=\"logo.png\" #{alt_attribute} width=\"407\">" }
+          let(:alt_attribute) { "alt=\"Logo alt attribute\"" }
+          let(:body) { { "en" => "<h2>This is my \"heading 2\" title</h2><h2><br></h2><p>A \"normal\" description below Heading 2</p><h3>Now this is my \"heading 3\"</h3><h3><br></h3><ul><li>This is my first option</li><li>This is my second option</li><li>This is my third option</li></ul><p><br></p><p>And below an uploaded image</p><p>#{image}</p><p><br></p><p><code>Here is code block</code></p>" } }
+
+          it "serializes the body without HTML tags" do
+            expect(serialized).to include(body: { "en" => "----------------------------\nThis is my \"heading 2\" title\n----------------------------\n\nA \"normal\" description below Heading 2\n\nNow this is my \"heading 3\"\n--------------------------\n\n* This is my first option\n* This is my second option\n* This is my third option\n\nAnd below an uploaded image\n\nLogo alt attribute\n\nHere is code block" })
```

Comment:
It might be a bit easier to read if you formatted this something like:

```ruby
body = <<~TEXT
  ----------------------------
  This is my "heading 2" title
  ----------------------------

  A "normal" description below Heading 2

  Now this is my "heading 3"
  --------------------------

  * This is my first option
  * This is my second option
  * This is my third option

  And below an uploaded image

  Logo alt attribute

  Here is code block
TEXT
```

Reply:
I updated the spec file according to your suggestion, hope it responds to your expectations